www/barp08/S2/index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="cs">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <meta name="viewport" content="width=device-width,initial-scale=1">
+    <meta name="description" content="Pong.">
+    <meta name="keywords" content="pong, ping-pong, homebrew, 4IZ268">
+    <meta name="web_author" content="Pavel Barton">
+    <meta name="icon_author" content="Bogdan Rosu from Flat-icon">
+    <link rel="icon" href="images/favs/favicon-m.png" type="image/png">
+    <title>Pong</title>
+    <link rel="stylesheet" href="css/styles.css">
+    <link rel="stylesheet" type="text/css" media="screen" href="css/styles.css" />
+    <link href="https://fonts.googleapis.com/css?family=Merriweather" rel="stylesheet">
+
+</head>
+
+<body>
+    <div class="wrapper">
+        <div class="navigation">
+            <header>
+                <a href="index.html"> <img class="logo" src="images/logo_5_inverted.png" alt="Pavel Bartoň"></a>
+            </header>
+            <nav>
+                <ul class="navbar">
+                    <li><a href="index.html">Pongo Bongo</a></li>
+                </ul>
+            </nav>
+        </div>
+        <main>
+            <canvas id="myCanvas" width="480" height="320"></canvas>
+            <script src="pong.js"></script>
+            <a href="https://twitter.com/share?ref_src=twsrc%5Etfw" class="twitter-share-button" data-size="large" data-text="Worst pong ever #noHate" data-via="pavelbarton" data-show-count="false">Tweet</a>
```

Comment:
tohle bohuzel neni AJAX, v praci nevidim AJAX nikde. Musis si vybrat nejakou API, nastudovat ji a prijit na to jak s ni komunikovat z prohlizece.

Reply:
Rozumim, udelal jsem nasledujici:
- svou shitty sablonu jsem nahradil bootstrapem, kdyz to bylo povoleno
- je tam AJAX, jedna se o Github API
- pong jsem se inspiroval na YT to je pravda, je to shit, nevedel jsem ze to bude narocne udelat blby pong..